_posts/2016-11-28-twis-84.md

```diff
@@ -0,0 +1,65 @@
+
+### Planning and Status
+
+Our overall [roadmap](https://github.com/servo/servo/wiki/Roadmap) is available online and now includes the Q4 plans and tentative outline of some ideas for 2017. Please check it out and provide feedback!
+
+This week's status updates are [here](http://statusupdates.dev.mozaws.net/project/servo). This service is shutting down immediately, and we will have to move to a new status update mechanism next week.
+
+### Notable Additions
+
+ - kvark [added](https://github.com/servo/webrender/pull/604) GPU marker support to WebRender
+ - razican [implemented](https://github.com/servo/servo/pull/14383) `column-rule-style` for Stylo
+ - nox [migrated](https://github.com/servo/servo/pull/14381) the Servo build to cargo workspaces
+ - ms2ger [removed](https://github.com/servo/servo/pull/14360) the legacy network stack
+ - chenpighead [implemented](https://github.com/servo/servo/pull/14358) `webkit-text-fill-color` and `webkit-text-stroke` for Stylo
+ - mrobinson [added](https://github.com/servo/webrender/pull/590) an API for scrolling individual layers in WebRender
+ - heycam [implemented](https://github.com/servo/servo/pull/14340) `StyleSheet.disabled` in CSSOM
+ - stshine [fixed](https://github.com/servo/servo/pull/14341) scroll to top
+ - upsuper [implemented](https://github.com/servo/servo/pull/14330) access to `CSSStyleRule` for Stylo
+ - glennw [changed](https://github.com/servo/webrender/pull/583) the font platforms to work with device pixels directly
+ - emilio [improved](https://github.com/servo/rust-bindgen/pull/288) handling of derived types inside namespaces in rust-bindgen
+ - nox [switched](https://github.com/servo/servo/pull/14284) to the `FnvHasher` in `http_loader::load` to reduce load from the default hasher
+ - connorimes [fixed](https://github.com/servo/servo/pull/14272) energy profiling
+ - kvark [switched](https://github.com/servo/webrender/pull/569) the WebRender mask generator to work on tiles
+ - nical [added](https://github.com/servo/webrender/pull/566) reference counting to images in WebRender to avoid extra copies
+ - heycam [optimized](https://github.com/servo/servo/pull/14256) the rule tree's garbage collection to look at the free list size
+ - vvuk [compiled](https://github.com/servo/webrender/pull/563) the shaders into WebRender to avoid requirement of a res directory
+ - vvuk [added](https://github.com/servo/servo/pull/14249) a `-Z wr-record` to enable WebRender display list recording
+ - fitzgen [added](https://github.com/servo/rust-bindgen/pull/264) IR debug output to rust-bindgen
+ - emilio [added](https://github.com/servo/saltfs/pull/536) support for per-repo reviewers to homu
+ - manishearth [added](https://github.com/servo/servo/pull/14190) support for immutable CSSOM
```

Comment:
basically all of CSSOM actually, the rest is in https://github.com/servo/servo/pull/14241